apis/python/tests/test_indexer_dtatye_perf.py

```diff
@@ -0,0 +1,99 @@
+from time import perf_counter
```

Comment:
Do we want to start adding perf tests to pytest unit tests?  Without some sort of pass/fail criteria, they don't seem very useful.  Historically we have separated unit tests (function, correctness) from performance, and only the former were run in CI

CC: @johnkerl for any thoughts he may have

Reply:
@bkmartinjr I moved the perf benchmark outside the CI. Agree totally they should not be in the CI. Also I have correctness version of those tests (smaller memory footprint) in the CI.